pandas/core/strings.py

```diff
@@ -2837,7 +2837,20 @@ def fullmatch(self, pat, case=True, flags=0, na=np.nan):
 
     @copy(str_replace)
     @forbid_nonstring_types(["bytes"])
-    def replace(self, pat, repl, n=-1, case=None, flags=0, regex=True):
+    def replace(self, pat, repl, n=-1, case=None, flags=0, regex=None):
+        if regex is None:
+            if (
+                isinstance(pat, str)
+                and len(pat) > 1
+                and any(c in pat for c in ".+*|^$[](){}\\")
```

Comment:
A question mark as well? Or can this only occur in combination of one of the others?

Reply:
Good catch, added